utils/snappy-move-results/generate_metadata.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
```

Comment:
Thoughts on moving this logic as a function in common.sh?

Reply:
yes will do that